docker-for-mac/multi-arch.md

```diff
@@ -54,3 +54,162 @@ Notice that this time, the `uname -a` output shows `armv7l` and
+
+```
+~ ❯❯❯ mkdir test && cd test && cat <<EOF > Dockerfile
+FROM ubuntu
+RUN apt-get update && apt-get install -y curl
+WORKDIR /src
+COPY . .
+EOF
+```
+
+```
+~/test ❯❯❯ docker buildx build --platform linux/amd64,linux/arm64,linux/arm/v7 -t username/demo:latest --push .
+[+] Building 6.9s (19/19) FINISHED
+...
+ => => pushing layers                                                             2.7s
+ => => pushing manifest for docker.io/username/demo:latest                       2.2
+ ```
+
+>   **Notes:**
+>
+>  - The `--platform` flag informs buildx to generate Linux images for Intel 64-bit, Arm 32-bit, and Arm 64-bit architectures.
+>  - The `--push` flag generates a multi-arch manifest and pushes all the images to Docker Hub.
+
+Inspect the image using `imagetools`.
+
+```
+~/test ❯❯❯ docker buildx imagetools inspect username/demo:latest
+Name:      docker.io/username/demo:latest
+MediaType: application/vnd.docker.distribution.manifest.list.v2+json
+Digest:    sha256:2a2769e4a50db6ac4fa39cf7fb300fa26680aba6ae30f241bb3b6225858eab76
+
+Manifests:
+  Name:      docker.io/username/demo:latest@sha256:8f77afbf7c1268aab1ee7f6ce169bb0d96b86f585587d259583a10d5cd56edca
+  MediaType: application/vnd.docker.distribution.manifest.v2+json
+  Platform:  linux/amd64
+
+  Name:      docker.io/username/demo:latest@sha256:2b77acdfea5dc5baa489ffab2a0b4a387666d1d526490e31845eb64e3e73ed20
+  MediaType: application/vnd.docker.distribution.manifest.v2+json
+  Platform:  linux/arm64
+
+  Name:      docker.io/username/demo:latest@sha256:723c22f366ae44e419d12706453a544ae92711ae52f510e226f6467d8228d191
+  MediaType: application/vnd.docker.distribution.manifest.v2+json
+  Platform:  linux/arm/v7
+  ```
+
+  The image is now available on Docker Hub with the tag `username/demo:latest`. You can use this image to run a container on Intel laptops, Amazon EC2 A1 instances, Raspberry Pis, and on other architectures. Docker pulls the correct image for the current architecture, so Raspberry Pis run the 32-bit Arm version and EC2 A1 instances run 64-bit Arm. The SHA tags identify a fully qualified image variant. You can also run images targeted for a different architecture on Docker Desktop.
+
+  You can run the images using the SHA tag, and verify the architecture. For example, when you run the following on a macOS:
+
+ ```
+  ~/test ❯❯❯ docker run --rm docker.io/username/demo:latest@sha256:2b77acdfea5dc5baa489ffab2a0b4a387666d1d526490e31845eb64e3e73ed20 uname -m
+aarch64
+```
+
+```
+~/test ❯❯❯ docker run --rm docker.io/username/demo:latest@sha256:723c22f366ae44e419d12706453a544ae92711ae52f510e226f6467d8228d191 uname -m
+armv7l
+```
+
+In the above example, `uname -m` returns `aarch64` and `armv7l` as expected, even when running the commands on a native macOS developer machine.
+
+### Building a Python Flask web application
```

Comment:
This part doesn't make sense as you are building nothing if you change to github.com/username/helloworld.  This whole section should be removed.

Reply:
Removed the `Building a Python Flask web application` section